src/app/components/Typography/index.ts

```diff
@@ -52,3 +53,16 @@ export const labelStyle = css`
   line-height: 1.6rem;
   font-weight: normal;
 `;
+
+export const disabledStyle = css`
```

Comment:
can you add this to `linkStyle` and `decoratedLinkStyle` instead of making a new thing? i'm pretty sure those css blocks can have functions in them that look at props